PWGHF/TableProducer/candidateCreator3Prong.cxx

```diff
@@ -34,10 +34,23 @@ using namespace o2::aod::hf_cand_3prong;
 using namespace o2::constants::physics;
 using namespace o2::framework;
 
+// centrality selection estimators
+enum CentralityEstimator {
+  None = 0,
+  FT0A,
+  FT0C,
+  FT0M,
+  FV0A,
+  NCentralityEstimators
+};
```

Comment:
In a few other implementations in the HF code also other estimators are used, see https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/D2H/Tasks/taskDs.cxx#L36-L42 and https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/TableProducer/treeCreatorLcToPKPi.cxx#L232-L237 . Maybe is it worth to add more estimators here too?

Reply:
At the moment I would stay with the few ones that we agreed for the candidate creators